runtime/vm/FFITypeHelpers.hpp

```diff
@@ -25,29 +25,18 @@
 
 #include "ut_j9vm.h"
 #include "vm_internal.h"
-#ifdef J9VM_OPT_PANAMA
-#include "ffi.h"
-#endif /* J9VM_OPT_PANAMA */
+
```

Comment:
Please delete this file (there's nothing useful left, and the only file that included is being removed).